tests/test_planner/rrt_test.py

```diff
@@ -14,16 +15,25 @@ def test_rrt():
         ],
     ]
 
+    poly = PolygonEnv(Environment)
+    # poly.update(general_obstacles_list)
+
     for obstacles in general_obstacles_list:
-        obstacle_list = obstacles
+        poly.update(obstacles)
 
         # Instatiate rrt planner object
         my_tree = RRT(sample_area=(-5, 15), sampler=sampler, expand_dis=0.1)
-        path, node_list = my_tree.plan((1, 1), (10, 10), obstacle_list)
+        path, node_list = my_tree.plan((1, 1), (10, 10), poly)
```

Comment:
This will need to be modified since plan should only return a trajectory

Reply:
After seeing the other suggested changes, this becomes confusing. Should my plan return only 'path' and not 'node_list', or should the returned path be a Trajectory? If my plan is not returning 'node_list', the user won't be able to ever visualize the path, along with the env. Right now the user has an _option_ of visualizing the path by un-commenting that line. (I am assuming that the user will look at this test file for a sample of how his code should be) Other changes have been implemented.